policyengine_uk/parameters/gov/hmrc/tax_free_childcare/age/disability.yaml

```
@@ -0,0 +1,12 @@
+description: The HMRC department extends the tax-free childcare program eligibility to children with disabilities up to this age threshold.
```

And in other descriptions; I don't think `HMRC department` is typically used
```suggestion
description: HMRC extends the tax-free childcare program eligibility to children with disabilities up to this age threshold.
```